src/store/printer/getters.ts

```diff
@@ -648,7 +648,7 @@ export const getters: GetterTree<PrinterState, RootState> = {
 
       if (
         supportedTypes.includes(type) &&
-        (!filterByPrefix.includes(type) || !name.startsWith('_'))
+        ((!filterByPrefix.includes(type) || !name.startsWith('_')) || showHidden)
```

Comment:
Simplify

```suggestion
        (showHidden || !filterByPrefix.includes(type) || !name.startsWith('_'))
```

Reply:
yep, that works better. will also change the name as mentioned above

Reply:
The name here is fine, as this is the `getOutputs` method, so in this specific context it is obvious we are referring to Outputs!